sdk/core/azure-core/src/main/java/com/azure/core/implementation/AccessTokenCache.java

```diff
@@ -140,8 +140,8 @@ public AccessTokenCache(TokenCredential tokenCredential) {
                         return sinksOne.asMono().switchIfEmpty(Mono.defer(() -> Mono.just(cache)));
                     }
                 }
-            } catch (Throwable t) {
-                return Mono.error(t);
+            } catch (Exception ex) {
+                return Mono.error(ex);
```

Comment:
Same as before - should this be monoError instead?

Reply:
I believe there was a reason the `AccessTokenCache` doesn't log this, maybe @g2vinay knows more